advanced/Scripts/version.sh

```diff
@@ -8,179 +8,93 @@
 # This file is copyright under the latest version of the EUPL.
 # Please see LICENSE file for your rights under this license.
 
-# Variables
-DEFAULT="-1"
-COREGITDIR="/etc/.pihole/"
-WEBGITDIR="/var/www/html/admin/"
-
 # Source the setupvars config file
 # shellcheck disable=SC1091
-source /etc/pihole/setupVars.conf
+. /etc/pihole/setupVars.conf
 
-getLocalVersion() {
-    # FTL requires a different method
-    if [[ "$1" == "FTL" ]]; then
-        pihole-FTL version
-        return 0
-    fi
+# Source the versions file poupulated by updatechecker.sh
+cachedVersions="/etc/pihole/versions"
 
-    # Get the tagged version of the local repository
-    local directory="${1}"
-    local version
+if [ -f ${cachedVersions} ]; then
+    # shellcheck disable=SC1090
+    . "$cachedVersions"
+else
+    echo "Could not find /etc/pihole/versions. Exiting."
```

Comment:
I think we should just have this run `/opt/pihole/updatechecker.sh` in the case of the versions file not existing. If that requires that the version script go back behind a root check - then so be it

Reply:
I run the `updatechecker` via `pihole updatechecker` which will check for root powers.

```
rockpi@rockpi-4b:~$ ls -lha /etc/pihole/versions
ls: cannot access '/etc/pihole/versions': No such file or directory
rockpi@rockpi-4b:~$ pihole -v
Could not find /etc/pihole/versions. Running update now.
[sudo] password for rockpi: 
  Pi-hole version is simplify_versions.sh v5.12.2-53-g52283478 (Latest: v5.13)
  AdminLTE version is 99 v5.16-5-g8abfde2e (Latest: v5.16)
  FTL version is fix/socket_warning vDev-bcdf653 (Latest: v5.18.2)


```